example-app/SantokuApp/src/features/app-updates/components/AppUpdatesChecker.tsx

```diff
@@ -5,7 +5,7 @@ import {useAppUpdates} from '../services/useAppUpdates';
 import {showUpdateRequiredDialog} from '../use-cases/showUpdateRequiredDialog';
 
 const AppUpdatesChecker: React.FC = ({children}) => {
-  const {data: appUpdates} = useAppUpdates();
+  const {appUpdates} = useAppUpdates();
```

Comment:
useAppUpdates is a query.
So it returns results, query status, etc.
I think it would be more appropriate to rename the hook to useAppUpdatesQuery.
What do you think?

Reply:
LGTM:+1:

By the way, I think that we can return only the `data`. It's normal that "query returns data," so it would be easy to understand too. And the caller could decide what to call the returned data.

```
const {data: appUpdate} = useAppUpdatesQuery();
```

But, I think it is a minor detail, so we can change it to what you suggested, or I don't think it will be a problem if we leave it unchanged.